app/migrations/Version20250415142826.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Mautic\CoreBundle\Doctrine\PreUpAssertionMigration;
+
+final class Version20250415142826 extends PreUpAssertionMigration
+{
+    protected const TABLE_NAME = 'campaign_lead_event_log';
+
+    protected function preUpAssertions(): void
+    {
+        $this->skipAssertion(
+            fn (Schema $schema) => $schema->getTable($this->getPrefixedTableName(self::TABLE_NAME))->hasIndex('{$this->prefix}idx_scheduled_events'),
```

Comment:
```suggestion
            fn (Schema $schema) => $schema->getTable($this->getPrefixedTableName(self::TABLE_NAME))->hasIndex("{$this->prefix}idx_scheduled_events"),
```
Looks like a typo, double quotes should be used here instead of single quotes.

Reply:
Great catch!